improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -303,12 +308,17 @@ def _interpolate_percentiles(
             forecast_at_percentiles, coord=percentile_coord_name
         )
 
-        (
-            original_percentiles,
-            forecast_at_reshaped_percentiles,
-        ) = self._add_bounds_to_percentiles_and_forecast_at_percentiles(
-            original_percentiles, forecast_at_reshaped_percentiles, bounds_pairing
-        )
+        if not self.skip_ecc_bounds:
+            cube_units = forecast_at_percentiles.units
+            bounds_pairing = get_bounds_of_distribution(
```

Comment:
Scientifically, probably not, but for the hot-fix we are keen to minimise the change as much as possible. Doing it this way means we can target the fix on just the tasks that have issues rather than risk unintended consequences.